client/templates/website-template/snowpack.config.cjs

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+  workspaceRoot: "../../modules/",
+  mount: {
+    src: "/",
+    public: {
+      url: "/",
+      static: true,
+      resolve: false
+    }
+  },
+  packageOptions: {
+    polyfillNode: true,
+    packageLookupFields: ["svelte"]
```

Comment:
It's not - it's a generic configuration option. All this does is add another field for Rollup to check for package dependencies. The editor already uses Svelte and I already had eslint configured for it.

Reply:
Right, but this is the template package

Reply:
I uh... I just don't see the need, lol. I was even willing to expand this template if we added other file types, so that there was a common source of configuration "knowledge"